README.md

```diff
@@ -102,7 +102,7 @@ following resources are available for you:
 
 - [Knative Users](https://groups.google.com/forum/#!forum/knative-users)
 - [Knative Developers](https://groups.google.com/forum/#!forum/knative-dev)
-- [Knative Slack](./SLACK-GUIDELINES.md). Ping @serving-help or @eventing-help
+- [Knative on CNCF Slack](./SLACK-GUIDELINES.md). Ping @serving-help or @eventing-help
```

Comment:
@ReToCode unfortunately we don't have these groups anymore. Maybe these can be removed or changed to #knative-serving and #knative-eventing channel references. IMO, they don't really need to be called out at all since it's odd to only mention those two but not all of the other bits.

Reply:
👍 Agreed, I removed the block.